python/desc/imsim/camera_readout.py

```diff
@@ -256,25 +291,27 @@ def _add_read_noise_and_bias(self, amp_name):
         amp_name : str
             The amplifier name, e.g., "R22_S11_C00".
         """
-        amp_props = self.fp_props.get_amp(amp_name)
+        amp_info = self.camera_info.get_amp_info(amp_name)
         full_arr = self.amp_images[amp_name].getArray()
-        full_arr += np.random.normal(scale=amp_props.read_noise,
+        full_arr += np.random.normal(scale=amp_info.getReadNoise(),
                                      size=full_arr.shape)
-        full_arr += amp_props.bias_level
+        full_arr += config['electronics_readout']['bias_level']
 
     def _apply_crosstalk(self):
```

Comment:
I think I would put this after _apply_crosstalk so it is in the same order of the operations. That will also make the docstrings a little easier to understand.

Reply:
So are you suggesting that the read noise and bias levels should be executed after the crosstalk is applied?  That's consistent with what Robert has said the order should be, but that's not in fact how the correction is applied in `ip_isr` (afaict): https://github.com/lsst/ip_isr/blob/master/python/lsst/ip/isr/isrTask.py#L540
Since the bias levels, we've been using are so low (1000 adu), it's a negligible effect, so we could apply the crosstalk first, but I'd still like some clarity on what's going on in `ip_isr`.

Reply:
Sorry... I wasn't suggesting changing the order of the operations. I was suggesting change the order of these functions in the file so that they *matched* the order of the operations.  When I was reading it I got confused because of this.